Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -61,15 +46,19 @@ open class ChatChannelListVC<ExtraData: UIExtraDataTypes>: UIViewController,
     ) -> UICollectionViewCell {
         let channel = controller.channels[indexPath.row]
         
-        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath)
+        let cell = collectionView.dequeueReusableCell(
+            withReuseIdentifier: "Cell",
+            for: indexPath
+        ) as! ChatChannelListCollectionViewCell<ExtraData>
         
-        if let channelView = cell.contentView.subviews.first as? ChatChannelView<ExtraData> {
+        if let channelView = cell.channelView {
             channelView.channel = channel
         } else {
-            let channelView = uiConfig.channelList.channelView.init(channel: channel, uiConfig: uiConfig)
-            cell.contentView.embed(channelView)
+            cell.channelView = uiConfig.channelList.channelView.init(channel: channel, uiConfig: uiConfig)
```

Comment:
Yeah, this shouldn't be needeed. The channel view should always be there. What is the reason for having the cell without its content?

Reply:
I've taken the cell from git stash as it was.

Pavel L. took the task about container cell and working on it right now but it looks like we have to stay synchronized on that topic.